mmdet3d/apis/test.py

```diff
@@ -29,8 +35,38 @@ def single_gpu_test(model, data_loader, show=False, out_dir=None):
             result = model(return_loss=False, rescale=True, **data)
 
         if show:
-            model.module.show_results(data, result, out_dir)
+            if hasattr(model.module, 'show_results'):
+                model.module.show_results(data, result, out_dir)
```

Comment:
`show_results` and `show_result` are so confusing here. Please at least add a comment to explain the former is for visualization in mmdet3d (maybe multi-modality visualization in the future).